connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -166,6 +169,13 @@ public void destroyConnector(final @PathParam("connector") String connector) thr
         completeOrForwardRequest(cb, "/connectors/" + connector, "DELETE", null);
     }
 
+    @PUT
+    @Path("/{connectorType}/config/validate")
```

Comment:
Yeah, lets do that -- we'll need it for https://issues.apache.org/jira/browse/KAFKA-3316 too since `GET /connectors/` is already used to list connector instances. I'm not a fan of having to including a dash or underscore in a resource name, but I guess that might be unavoidable here. We could reduce it just to `plugins`, though that may not be specific enough (e.g. there are also Converter plugins).